providers/catalog.go

```diff
@@ -2147,4 +2148,23 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Recurly Basic authentication
+	Recurly: {
+		AuthType: Basic,
+		BaseURL:  "https://v3.recurly.com",
```

Comment:
@laurenzlong Interesting position of the API version. We might have to just hardcode this one.

Reply:
yeah agreed